programs/vote/src/vote_state/vote_state_0_23_5.rs

```diff
@@ -0,0 +1,60 @@
+use super::*;
+
+const MAX_ITEMS: usize = 32;
```

Comment:
related to [this](https://github.com/solana-labs/solana/pull/8380/files#r382686079), if these are expected order of numbers, `u8` is just enough?

Reply:
@ryoqun this is copy-pasted from the old version of the vote state, trying to stay as close to the original as possible.